src/views/Home.jsx

```diff
@@ -26,60 +31,109 @@ export function Home({ handleListTokenState }) {
 		if (placeholder) {
 			handleListTokenState(newToken);
 		} else {
-			setMessage('Error adding empty list to Firebase');
+			toast({
+				title: 'Error',
+				description: 'Error adding empty list to Firebase',
+				variant: 'errorToast',
+			});
 		}
 	};
 
 	const handleChange = (e) => setUserEnteredToken(e.target.value);
 	const handleFormSubmit = (e) => {
 		e.preventDefault();
 		if (userEnteredToken.length === 0) {
-			return setMessage('Please enter a token.');
+			toast({
+				title: 'Error',
```

Comment:
Should've been clear about this, but we are leaving out the title for the toast on list view. We should try to style all the toasts across the app the same way. Just thought it looks clunky with it. What do you think @katherineyuneman? If we want to keep the title then we should edit our branch.